scripts/utils/trading_strategy_helpers.js

```diff
@@ -294,15 +291,17 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     return buildBundledTransaction(transactions)
   }
 
-  const calculateBuyAndSellAmountsFromPrice = function(price, targetToken) {
-    // Sell targetToken for stableToken at price with unlimited orders
-    // Example:
-    // Sell 1 ETH at for 102 DAI (unlimited)
-    // Sell x ETH for max256 DAI
-    // x = max256 / 102
-    // priceFormatted = 102000000000000000000
-    price = price.toFixed(18)
-    const priceFormatted = toErc20Units(price, targetToken.decimals)
+  const calculateBuyAndSellAmountsFromPrice = function(price, stableToken, targetToken, maximalRoundingErrorInPercent = 0.1) {
+    // decimalsForPrice: Decimals of reference price (OWL Price) + stableToken.decimals - targetToken.decimals
+    const decimalsForPrice = 18 + stableToken.decimals - targetToken.decimals
+    if (decimalsForPrice < 0 || decimalsForPrice > 36) {
+      throw "decimalsForPrice is not reasonable. Don't apply the bracket-strategy for this token pair!"
```

Comment:
Test for these errors will come in a separate PR, so that we can continue testing for now.